README.md

```diff
@@ -46,18 +75,22 @@ Supported Distributions
 * Ubuntu
 * Debian 10+ (ipaclient only, no server or replica!)
 
+
 Requirements
 ------------
 
 **Controller**
 * Ansible version: 2.8+ (ansible-freeipa is an Ansible Collection)
-* /usr/bin/kinit is required on the controller if a one time password (OTP) is used
-* python3-gssapi is required on the controller if a one time password (OTP) is used with keytab to install the client.
+* /usr/bin/kinit is required on the controller if a one time password
+(OTP) is used
+* python3-gssapi is required on the controller if a one time password
+(OTP) is used with keytab to install the client.
```

Comment:
Why are these lines split?

Reply:
Wrong formatting, they shouldn't change. I'll fix it.